### PR TITLE
Simplify Solvers

### LINKED ISSUE #165: Minimize number of solvers for testing
We currently have:

SCS
Ipopt
OSQP
GLPK
Clp

We should choose between Ipopt and OSQP, and between GLPK and Clp.

Or we could even just use SCS and Ipopt.

### LINKED ISSUE #172: Add new MOI tests

### LINKED ISSUE #182: Add `;`'s in Thermal_Generation_Dispatch_Example

### PR DESCRIPTION
**Pull request by joaquimg**
Close #165
Close #172 
Close #182

### CODE REVIEW

**Comment by joaquimg** [comment]
MOI Unit tests commented out:
```julia
                "test_model_copy_to_UnsupportedAttribute",
                "test_conic_linear_VectorOfVariables_2",
                "test_attribute_SolverVersion",
```

Suggestions for solver version? inner solver? diffopt version? both?
ex: `"DiffOpt=x.y.z with $(solvername)=a.b.c"`?

**Comment by joaquimg** [comment]
@blegat and @matbesancon, for me we can tag a new version as soon as we merge this. Also probably fine merging even with those two excludes.

**Review by matbesancon — APPROVED** [review]

**Comment by matbesancon** [comment]
I'll let @blegat take a look at the last error before merging. The SolverVersion attribute is not urgent

**Review by blegat — APPROVED** [review]

**Pull request merged**
